packages/cli/src/templates/index.ts

```diff
@@ -0,0 +1,27 @@
+import { existsSync } from 'fs';
+import { mkdir, writeFile } from 'fs/promises';
+import { htmlTemplate } from './html';
+import { onePageTemplate } from './one-page';
+
+type FileOptions = {
+  snakeCaseName: string;
+  name: string;
+};
+
+export const writeBaseWebpackProjectFiles = async ({
+  snakeCaseName,
+  name
+}: FileOptions) => {
+  if (existsSync(snakeCaseName)) {
```

Comment:
Idk why this threw me off a bit, but this is basically an implicit relative path, right? Wonder if it'd be more readable to have something like:

```ts
const outPath = path.resolve(process.cwd(), snakeCaseName);
```

and then use that throughout this fn.